python/unitytrainers/models.py

```diff
@@ -229,10 +242,10 @@ def create_cc_actor_critic(self, h_size, num_layers):
         a = tf.exp(-1 * tf.pow(tf.stop_gradient(self.output) - self.mu, 2) / (2 * self.sigma_sq))
         b = 1 / tf.sqrt(2 * self.sigma_sq * np.pi)
         self.all_probs = tf.multiply(a, b, name="action_probs")
-        self.probs = tf.identity(self.all_probs)
+        self.probs = tf.reduce_prod(self.all_probs, axis=1)
```

Comment:
Are you testing things out are do we go with reduce_prod ?

Reply:
We will keep it for now. It is theoretically correct, and balance ball trains very fast.